assets/panel/debugger.properties

```diff
@@ -146,7 +146,7 @@ sourceSearch.search.again.key=G
 
 # LOCALIZATION NOTE (sourceSearch.resultsSummary1): Shows a summary of
 # the number of matches for autocomplete
-sourceSearch.resultsSummary1=%d results
+sourceSearch.resultsSummary1=%d of %d results
```

Comment:
Maybe just use `editor.searchResults` property since it's already formatted this way?